src/Run/Setup.cs

```diff
@@ -77,40 +121,83 @@ private string FindSettingType(string valueToFind)
+                                               ).ToDictionary(s => s.Key, s => string.Empty) ?? new Dictionary<string, string>();
+            // A dev may have overriden the default values for a tool setting, but not specified the ValueType
+            foreach(var key in ToolSettings.Keys)
+            {
+                Settings[key].ValueType = "runToolSetting";
+            }
+        }
+
+        private void SetRunToolSettingsDefaults()
+        {
+            // If RunQuiet is already defined in config.json, don't override it
+            if (!Settings.ContainsKey(runQuietReservedKeyword))
+            {
+                Setting runQuietSetting = new Setting()
+                {
+                    Values = new List<string>() { "True", "False" },
+                    ValueType = "runToolSetting",
+                    Description = "Run tool specific setting.  Set to True to only display output from the executing command.  Default, False",
+                    DefaultValue = "false"
+                };
+
+                Settings.Add(runQuietReservedKeyword, runQuietSetting);
+            }
+
         }
 
         public int ExecuteCommand(string commandSelectedByUser, List<string> parametersSelectedByUser)
         {
+            ParseRunToolSettings(commandSelectedByUser);
+            string runQuietValue;
+            bool runQuiet = false;
+            if (ToolSettings.TryGetValue(runQuietReservedKeyword, out runQuietValue))
+            {
+                runQuiet = runQuietValue.Equals("true", StringComparison.OrdinalIgnoreCase) ? true : false;
+            }
             CompleteCommand commandToRun = BuildCommand(commandSelectedByUser, parametersSelectedByUser);
             if (commandToRun != null)
             {
-                Console.ForegroundColor = ConsoleColor.DarkYellow;
-                Console.WriteLine("Running: {0} {1}", commandToRun.ToolCommand, commandToRun.ParametersCommand);
-                Console.ResetColor();
-
-                int result = RunProcess.ExecuteProcess(commandToRun.ToolCommand, commandToRun.ParametersCommand);
-                if (result == 0)
+                if (!runQuiet)
                 {
-                    Console.ForegroundColor = ConsoleColor.Green;
-                    Console.WriteLine("Build Succeeded.");
+                    PrintColorMessage(ConsoleColor.DarkYellow, "Running: {0} {1}", commandToRun.ToolCommand, commandToRun.ParametersCommand);
                 }
-                else
+
+                int result = RunProcess.ExecuteProcess(commandToRun.ToolCommand, commandToRun.ParametersCommand);
+                if (!runQuiet)
                 {
-                    Console.ForegroundColor = ConsoleColor.Red;
-                    Console.WriteLine("Build Failed.");
+                    if (result == 0)
+                    {
+                        PrintColorMessage(ConsoleColor.Green, "Build Succeeded.");
```

Comment:
Yes something like that, another thing that I find useful for failed builds is also the exit code. So on failures it might be useful to say `Command execution failed with exit code {0}`.